src/glossary/_index.md

```diff
@@ -228,6 +228,24 @@ A sector is a contiguous array of bytes that a miner puts together, seals, and p
 
 ## Slashing
 
+It is important to understand that Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
```

Comment:
```suggestion
Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
```

Reply:
Probably hundreds of these very important details so suggesting this simplification. Not a blocking comment.